.github/workflows/ci.yml

```diff
@@ -6,21 +6,10 @@ on:
     - master
   pull_request:
 jobs:
-  test:
-    strategy:
-      matrix:
-        os: ['ubuntu-20.04', 'macOS-11', 'windows-2019']
-        go: ['1.18.x', '1.17.x']
-    runs-on: ${{ matrix.os }}
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
Though `runs-on: ubuntu` don't check sources that is targeted to other GOOS/GOARCH, it is ok because this package contains only portable Go sources.